packages/did-comm/src/__tests__/did-comm.action-handler.test.ts

```diff
@@ -0,0 +1,150 @@
+import {
```

Comment:
could we exclude this file from the PR since the changes are not final yet?

Reply:
I will delete it since it wasn't actually testing did-comm related code